packages/dds/merge-tree/src/mergeTree.ts

```diff
@@ -837,7 +844,7 @@ export class MergeTree {
             offset = start;
             return false;
         };
-        this.searchBlock(this.root, pos, 0, refSeq, clientId, { leaf }, undefined, localSeq);
+        this.nodeMap(refSeq, clientId, leaf, undefined, undefined, pos, pos + 1, localSeq);
```

Comment:
search block was the wrong method here, as it stops at every segment, whereas nodemap uses the tree to optimally find the position.

Reply:
What consequences did that have? perf?

Reply:
walking the tree is with node walk is O(log N), whereas walking all segments with search will be O(N), so yeah it was perf